### PR TITLE
Collapsed Dropdown to a long list.

### LINKED ISSUE #1553: Changelog units drop down isn't discoverable
Feedback from the beta forums is that the units changes aren't easily found. They are hidden behind a drop down and can't be found in a browser search.

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #1553

### CODE REVIEW

**Comment by Daltz333** [comment]
Can we move the list to its own section on the very bottom and just add a link where it previously was?

The long list provides no benefit to most teams (Java and LabVIEW) and obscures valuably information.

**Comment by jasondaming** [comment]
I disagree.  I think this whole document is full of things that don't apply to most teams.  This list takes up the equivalent space of three lines that isn't that much space.  It doesn't obscure anything more than this entire document was already obscure but it serves as a reference that teams can quickly scan.

**Comment by Daltz333** [comment]
It does not hurt to just simply have a link toward the bottom of the page for relevant breaking classes, while it only benefits teams that are not C++ by making it easier to read relevant information.

**Comment by jasondaming** [comment]
What I am saying is that there is no reason to give this list special treatment vs any of the other things.  You are just way over thinking it.

**Comment by jasondaming** [comment]
@sciencewhiz be the arbiter?

**Comment by sciencewhiz** [comment]
I think it's reasonable, it's not that much longer then other entires

File: source/docs/yearly-overview/yearly-changelog.rst [hunk 1]
```diff
@@ -66,38 +66,7 @@ Breaking Changes
...
-   - ``InterruptableSensorBase``
-   - ``MotorSafety``
-   - ``Notifier``
-   - ``SPI``
-   - ``SerialPort``
-   - ``SlewRateLimiter``
-   - ``Solenoid``
-   - ``Timer``
-   - ``Watchdog``
+- C++: Various classes have migrated to use units: ``Ultrasonic``, ``CommandScheduler``,   - ``CommandState``, ``WaitUntilCommand``, ``MecanumControllerCommand``, ``RamseteCommand``, ``SwerveControllerCommand``, ``TrapezoidProfileCommand``, ``WaitCommand``, ``Command`` (Old Commands), ``CommandGroup`` (Old Commands), ``CommandGroupEntry`` (Old Commands), ``TimedCommand`` (Old Commands), ``WaitCommand`` (Old Commands), ``WaitForChildren`` (Old Commands), ``WaitUntilCommand`` (Old Commands), ``Counter``, ``CounterBase``, ``DriverStation``, ``Encoder``, ``InterruptableSensorBase``, ``MotorSafety``, ``Notifier``, ``SPI``, ``SerialPort``, ``SlewRateLimiter``, ``Solenoid``, ``Timer``, ``Watchdog``
```
Comment (sciencewhiz) [inline source/docs/yearly-overview/yearly-changelog.rst:69]:
```suggestion
- C++: Various classes have migrated to use units: ``Ultrasonic``, ``CommandScheduler``, ``CommandState``, ``WaitUntilCommand``, ``MecanumControllerCommand``, ``RamseteCommand``, ``SwerveControllerCommand``, ``TrapezoidProfileCommand``, ``WaitCommand``, ``Counter``, ``CounterBase``, ``DriverStation``, ``Encoder``, ``InterruptableSensorBase``, ``MotorSafety``, ``Notifier``, ``SPI``, ``SerialPort``, ``SlewRateLimiter``, ``Solenoid``, ``Timer``, ``Watchdog``, and Old Commands: ``Command``, ``CommandGroup``, ``CommandGroupEntry``, ``TimedCommand``, ``WaitCommand``, ``WaitForChildren``, ``WaitUntilCommand``, 
```

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**